### ISSUE TITLE
Support for `ordered` option with eqJoin

### ISSUE THREAD

**Issue by patrickfatrick** [neumino/rethinkdbdash#249]
RethinkDB 2.3 introduced the `ordered` option for eqJoin since it now defaults to not respecting order. With Thinky v2.3.2/rethinkdbdash v2.3.4 I get an error when trying to use it,

```
ReqlDriverError: Unrecognized option `ordered` in `eqJoin` after:
  r.row("tasks")
  Available option is index <string>
```

Thanks for any help!

**Comment by patrickfatrick** [comment]
You're a machine! Thanks for hopping on that so quick.

**Comment by neumino** [comment]
Fixed in 2.3.5
